packages/lesswrong/components/recentDiscussion/RecentDiscussionSubscribeReminder.tsx

```diff
@@ -251,14 +274,18 @@ const RecentDiscussionSubscribeReminder = ({classes}: {
         {dontAskAgainButton}
       </div>
     </AnalyticsWrapper>
-  } else if (!currentUser.emailSubscribedToCurated || adminBranch===3) {
+  } else if (!currentUserSubscribed || adminBranch===3) {
     // User is logged in, and has an email address associated with their
     // account, but is not subscribed to curated posts. A Subscribe button which
     // sets the subscribe-to-curated option, and (if their email address isn't
     // verified) resends the verification email.
+    const subscribeText = forumTypeSetting.get() === 'EAForum' ?
+      'Subscribe to our weekly EA Forum Digest email!' :
```

Comment:
Here's an updated version using Ben's suggested text (and a larger button font size):
<img width="500" alt="Screen Shot 2021-09-09 at 10 18 32 PM" src="https://user-images.githubusercontent.com/9057804/132764042-78503013-046a-4079-9596-7bbe0f2ee442.png">

Personally I'm rather allergic to blocks of text, and my experience with designers gives me the impression that people don't read explanations. But I'm sympathetic to wanting an explanation, and since both you and Ben suggested it I figured we should have one. Let me know if you have thoughts on this version.

Reply:
Also updated the logged out version to match:
<img width="500" alt="Screen Shot 2021-09-09 at 10 24 27 PM" src="https://user-images.githubusercontent.com/9057804/132764765-c826bdc8-5d55-44e6-a922-c84f4ca298ac.png">

Reply:
Yeah IMO that’s way too much text, but I’ll let you and Ben decide that. I’d just say like, “A curated selection of posts sent out once a week by Aaron Gertler.”